### PR TITLE
Feature1

### PR DESCRIPTION
**Pull request by Iptysam**
- Add index.html file
- Add style.css file
- Update a README.md file
- Add linters

### CODE REVIEW

**Review by Reem-lab — COMMENTED** [review]
# Hi @Iptysam, 👋🏻👋🏻 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't contain any linters files jindly add hintrc to your project and stylelint as well and please make all your changes in a new branch since I can't see any files for `hello microverse` heading in feature1 branch.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: .hintrc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "formatters": ["stylish"],
+    "hints": [
+      "button-type",
+      "disown-opener",
+      "html-checker",
+      "meta-charset-utf-8",
+      "meta-viewport",
+      "no-inline-styles:error"
+    ]
+  }
```
Comment (Reem-lab) [inline .hintrc:18]: - [ ] Great on set up the linters files but I think to install the packages for hintrc and stylelint if you scroll down [here](https://github.com/microverseinc/linters-config/tree/master/html-css) you will see the steps please add it to your project ✅ and please don't forget after installing the packages to put `node_modules` in gitignore file 🚀🚀
image

**Review by Reem-lab — COMMENTED** [review]
# Hi @Iptysam, 👋🏻👋🏻 

Good job so far! 🔥🔥🚀🚀
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file. ✅

## Required Changes ♻️
_Check the comments under the review._
- [ ] I think you forget to create a test.md file as required so kindly create it and write the required paragraph inside it. 
image


 - [ ] I suggest adding the `MIT.md` file to your project :ok_hand:, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it :wink:
>
> > The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. :100:
>

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bondok6 — COMMENTED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @Iptysam 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- No Linter errors ✅
- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

- [OPTIONAL]  Great job adding `LICENSE.md` file to your project 💯 However, the LICENSE Link in your README is broken, so Kindly fix it 👌

> 💡 To fix this part go to your README and update it (Write LICENSE.md instead of LICENSE)
>  should be like this
``` This project is [MIT](./LICENSE.md) licensed. ```

- [OPTIONAL] When I'm trying to approve this PR, I got this message [I can't access the approve button]. I think you should edit something in your Settings to fix this problem. For now, I'll leave a comment 👍




_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**